modules/diskq/qdisk.c

```diff
@@ -213,6 +213,13 @@ _is_free_space_between_write_head_and_backlog_head(QDisk *self, gint msg_len)
   return self->hdr->write_head + msg_len < self->hdr->backlog_head;
 }
 
+static inline gboolean
+_is_free_space_at_the_beginning_of_qdisk(QDisk *self, gint msg_len)
```

Comment:
[optional]

The function name feels weird a bit, can you consider `_is_there_free_space_at_the_beginning_of_qdisk()`?

Reply:
This follows the name of another function: `_is_free_space_between_write_head_and_backlog_head()`. I can rename both.